swupd/manifest.go

```diff
@@ -580,8 +580,10 @@ func linkDeltaPeersForPack(c *config, oldManifest, newManifest *Manifest) error
 	}
 
 	// Run rename detection on old and new manifests
-	renameDetection(newManifest, added, removed, *c)
-
+	err := renameDetection(newManifest, added, removed, *c)
+	if err != nil {
+		return err
+	}
```

Comment:
you can just call `return renameDetection(...)` instead.

Reply:
Updated.